.github/workflows/pr.yml

```diff
@@ -338,20 +338,20 @@ jobs:
 
       - name: Install dependencies (npm ci)
         run: npm ci --prefer-offline
-      
-      - name: Install Dot.net	
-        uses: actions/setup-dotnet@v1	
-        with:	
-          dotnet-version: ${{env.DOTNET_VERSION}}	
-        if: matrix.test-suite != 'notebookWithoutPythonExt'	
 
-      - name: Install .NET Interactive	
-        run: dotnet tool install -g --add-source "https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json" Microsoft.dotnet-interactive	
-        if: matrix.test-suite != 'notebookWithoutPythonExt'	
+      - name: Install Dot.net
```

Comment:
Not sure why this is here, but leave it (no harm cause).
Difficult with all of the copy paste going on in github actions - problem is with github actions.